java/DEVELOPER.md

```diff
@@ -242,6 +242,13 @@ To run one class, use the following command:
 ./gradlew :client:test --tests 'TransactionTests' --rerun
 ```
 
+IT suite start the server for testing - standalone and cluster installation using `cluster_manager` script.
+If you want IT to use already started servers, use the following command line:
```

Comment:
Describe this as a command-line option.  If they want to know the default, they can look at the code.  
```
To run IT tests against an existing cluster or standalone endpoint, use: 
```